core/src/main/scala/io/circe/sbt/CirceOrgPlugin.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2022 circe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.circe.sbt
+
+import org.typelevel.sbt._
+import org.typelevel.sbt.gha._
+import sbt.Keys._
+import sbt._
+import scalafix.sbt.ScalafixPlugin.autoImport._
+import scoverage.ScoverageSbtPlugin.autoImport._
+import scoverage._
+
+import GenerativeKeys._
+import TypelevelSettingsPlugin.autoImport._
+import TypelevelCiPlugin.autoImport._
+import TypelevelSonatypePlugin.autoImport._
+
+object CirceOrgPlugin extends AutoPlugin {
+  object autoImport {
+    lazy val circeRootOfCodeCoverage = settingKey[Option[String]](
+      "None if code coverage is disabled. Defined with name of the root project from which to run code coverage."
+    )
+  }
+
+  import autoImport._
+
+  override def trigger = allRequirements
+
+  override def requires: Plugins = TypelevelPlugin && ScoverageSbtPlugin
+
+  override def buildSettings: Seq[Setting[_]] =
+    publishSettings ++ organizationSettings ++ scalafixSettings ++ githubActionsSettings
+
+  override def projectSettings = Seq(
+    coverageHighlighting := true,
+    coverageExcludedPackages := "io.circe.examples.*"
```

Comment:
I wonder, can these be moved to `buildSettings`? In case a project needs to override them for some reason it's always easier to override build-level settings. But depends on how they are defined in sbt-scoverage.

Reply:
Moved!